tests/test_brm.py

```diff
@@ -1043,7 +1043,8 @@ def test_coef_names(formula_str, non_real_cols, expected_names):
 @pytest.mark.parametrize('fitargs', [
     lambda S: dict(backend=pyro_backend, algo='prior', num_samples=S),
     lambda S: dict(backend=pyro_backend, iter=S, warmup=0),
-    lambda S: dict(backend=pyro_backend, iter=S // 2, num_chains=2, warmup=0),
+    pytest.param(lambda S: dict(backend=pyro_backend, iter=S // 2, num_chains=2, warmup=0),
+                 marks=pytest.mark.xfail('CI' in os.environ, reason='https://github.com/pyro-ppl/pyro/issues/2095')),
```

Comment:
This should rarely happen in practice, but on CI where we have just 1 or 2 cores, Pyro changes the number of chains to number of available cores, but doesn't change the number of samples to `num_samples * num_chains`